.github/workflows/containers-and-az-pool.yaml

```diff
@@ -79,9 +79,10 @@ jobs:
       POOL_ID: "cfa-epinow2-${{ needs.build-pipeline-image.outputs.tag }}"
       BATCH_ACCOUNT:     "cfaprdba"
       BATCH_ENDPOINT:    "https://cfaprdba.eastus.batch.azure.com/"
-      VM_IMAGE_TAG:      "canonical:0001-com-ubuntu-server-focal:20_04-lts"
-      NODE_AGENT_SKU_ID: "batch.node.ubuntu 20.04"
-      VM_SIZE:           "standard_a4m_v2"
+        # MPW: commenting out VM_IMAGE_TAG -> I don't see it being used
```

Comment:
Interesting, I'd never noticed this one before. But if we're sure it's not being used, then let's kick it